### PR TITLE
[DLFL21] Update 07-3.md

### PR DESCRIPTION
**Pull request by cswpy**
Here the instructions is confusing, do is a variable that refers to the dropout layers, should specify in the code.

### CODE REVIEW

File: docs/en/week11/11-2.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ Loss functional: $\mathcal{L} (E, S)$
 * Measures the quality of an energy function on training set
 * Invariant under permutations and repetitions of the samples.
 
-Training: $W^* = \min_{W\in \mathcal{W}} \mathcal{L}(W, S)$.
+Training: $W^* = \argmin_{W\in \mathcal{W}} \mathcal{L}(W, S)$.
```
Comment (Atcold) [inline docs/en/week11/11-2.md:157]: There is no `\argmin` function. You need to use `\arg\min`.

File: docs/en/week07/07-3.md [hunk 2]
```diff
@@ -308,7 +308,7 @@ Fig. 20 shows the output of the standard autoencoder.
 ### Train denoising autoencoder
 
 For denoising autoencoder, you need to add the following steps:<br>
-1) Calling `nn.Dropout()` to randomly turning off neurons. <br>
+1) Calling `do = nn.Dropout()` to randomly turning off neurons. <br>
```
Comment (Atcold) [inline docs/en/week07/07-3.md:311]:
```suggestion
1) Calling `do = nn.Dropout()` create a function that randomly turns off neurons. <br>
```

**Review by Atcold — CHANGES_REQUESTED** [review]
These edits need to be propagated to the French version (@lbourdois).

**Comment by cswpy** [comment]
Thanks for the review, I have updated the proposed changes.

**Review by Atcold — COMMENTED** [review]
You edited 92 files???

image

**Comment by cswpy** [comment]
Apologies, I accidentally pulled from the main repo before committing. Fixed now.

**Review by Atcold — APPROVED** [review]
Thanks, looks good.

**Comment by Atcold** [comment]
Actually, these should be propagated to *all* languages…
